extensions/amp-mustache/0.1/test/test-amp-mustache.js

```diff
@@ -424,6 +486,49 @@ describe('amp-mustache 0.1', () => {
     });
   });
 
+  describe('tables', () => {
+    it('should not foster text nodes in script template', done => {
+      if (isTemplateTypeScript) {
+        textContentSetup(
+            '<table>' +
+              '<tbody>' +
+                '<tr>' +
+                  '<td>Comment:</td>' +
+                  '<td>{{content}}</td>' +
+                '</tr>' +
+                '{{#replies}}' +
+                '<tr>' +
+                  '<td>Reply:</td>' +
+                  '<td>{{content}}</td>' +
+                '</tr>' +
+              '{{/replies}}' +
+              '</tbody>' +
+            '</table>');
```

Comment:
Can we use a more minimal example than this? It's also worth running the same example on the template case to highlight the difference.

Reply:
done. I removed some table data to simplify the example.